base_app/yelp_service.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import random
+import requests
+from base_app.restaurant import Restaurant
+from base_app.restaurant_schema import RestaurantSchema
+from base_app.photo import Photo
+from base_app.photo_schema import PhotoSchema
+from dotenv import load_dotenv
+load_dotenv()
+
+class YelpService:
+
+    def get_recommendation(self, params):
+        url = 'https://api.yelp.com/v3/businesses/search'
+        response = self.connection(url, params)
+        json_data = json.dumps(response.json())
+        restaurants = Restaurant.from_json(json_data)
+        restaurant = random.choice(restaurants)
+        schema = RestaurantSchema()
+        result = schema.dump(restaurant)
+        return result
+
+    def get_photos(self, restaurant_result):
+        restaurant_id = restaurant_result['data']['id']
+        restaurant_url = f'https://api.yelp.com/v3/businesses/{restaurant_id}'
+
+        response = self.connection(restaurant_url)
+        json_data = json.dumps(response.json())
+        photos = Photo.from_json(json_data)
+        schema = PhotoSchema()
+        result = schema.dump(photos)
+        return result
```

Comment:
These two methods could later be combined as the `/businesses/{id}` endpoint includes all restaurant-related data, including photos.

Reply:
We can do that tomorrow. We also have to refactor the random endpoint we merged in PR #38